app/src/main/java/com/keylesspalace/tusky/adapter/AccountSelectionAdapter.kt

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2019 Levi Bard
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.adapter
+
+import android.content.Context
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.ImageView
+import android.widget.TextView
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.db.AccountEntity
+import com.keylesspalace.tusky.util.CustomEmojiHelper
+import com.squareup.picasso.Picasso
+
+class AccountSelectionAdapter(context: Context): ArrayAdapter<AccountEntity>(context, R.layout.item_autocomplete) {
+    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+        var view = convertView
+
+        if (convertView == null) {
+            val layoutInflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+            view = layoutInflater.inflate(R.layout.item_autocomplete, parent, false)
+        }
+
+        val account = getItem(position)
+        if (account != null) {
+            val username = view!!.findViewById<TextView>(R.id.username)
```

Comment:
We probably won't have enough rows to recycle so implementing holder doesn't make too much sense, right?..

Reply:
nah I wouldn't bother either. But I would use Kotlin Android extension instead of `findViewById`